tenable/io/v3/vm/scanners.py

```diff
@@ -0,0 +1,364 @@
+"""
+scanners
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`scanners <scanners>` API.
+
+Methods available on ``tio.v3.vm.scanners``:
+
+.. rst-class:: hide-signature
+.. autoclass:: ScannersAPI
+
+    .. automethod:: control_scan
+    .. automethod:: delete
+    .. automethod:: details
+    .. automethod:: edit
+    .. automethod:: get_aws_targets
+    .. automethod:: get_scanner_key
+    .. automethod:: get_scans
+    .. automethod:: linking_key
+    .. automethod:: search
+    .. automethod:: toggle_link_state
+"""
+# from tenable.io.v3.base import TIOEndpoint
+from tenable.io.v3.base.endpoints.uw import UWBaseEndpoint
+
+
+class ScannersAPI(UWBaseEndpoint):
```

Comment:
attributes `_path` and `_conv_json` should be set

Reply:
Added